.template/addons/github/.github/workflows/review_code.yml.tt

```diff
@@ -1,58 +1,79 @@
 name: Review code
 
-on:
-  pull_request:
-    types: [opened, reopened, synchronize]
+on: pull_request
 
 env:
   DANGER_GITHUB_API_TOKEN: ${{ github.token }}
 
 jobs:
+  wait_for_test_build:
+    name: Wait for test build
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Wait for tests build
+        uses: lewagon/wait-on-check-action@v1.1.2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          check-name: 'Build Docker image'
+          running-workflow-name: 'Wait for test build'
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          wait-interval: 20
```

Comment:
I could not find an option, but is there a way to "Cancel the whole workflow, if the `Test` workflow failed"? 

Currently, the `review_code` workflow will "fail" if the `test` workflow failed. It would be clearer to have `review_code` canceled if `test` failed. 🤔 

Tho this is an optional improvement IMO :)

Reply:
@malparty I think we can use [allowed-conclusions param](https://github.com/lewagon/wait-on-check-action#allowed-conclusions) to allow the job to run only when the job it waits for is completed with success, you can see the demonstration on the updated PR description and for the code, I updated in [78c7421](https://github.com/nimblehq/rails-templates/pull/351/commits/78c7421579c0ce58a678720188f860b30348d4c3)

Reply:
It does not solve 100% my concern — i.e. having `code_review` cancelled (or `skipped`) instead of `failed`.
But as I cannot find alternatives at the moment, we can say it is good~